libs/arcade-core/arcade_core/api_wrapper/schema.py

```diff
@@ -0,0 +1,129 @@
+from typing import Literal
+
+from pydantic import BaseModel, Field
+
+from arcade_core.schema import InputParameter, ToolDefinition, ToolInput
+
+
+class ObjectMetadata(BaseModel):
+    """Object metadata (part of the serialized JSON stored in a Wrapper toolkit package)."""
+
+    object_type: str
+    """The type of the object."""
```

Comment:
I imagine that this 'type' can be an enum of known values?

Reply:
True, changed it to `Literal["http_endpoint", "api_wrapper_tool"]`